tests/unit/Form/Util/FieldNameObfuscatorTest.php

```diff
@@ -2,46 +2,43 @@
 
 namespace MailPoet\Form\Util;
 
-use Codeception\Stub;
 use MailPoet\WP\Functions as WPFunctions;
+use PHPUnit\Framework\MockObject\MockObject;
 
 class FieldNameObfuscatorTest extends \MailPoetUnitTest {
+
+  /** @var MockObject | WPFunctions */
```

Comment:
I think this should be an intersection type using `&` instead of `|` https://medium.com/@ondrejmirtes/union-types-vs-intersection-types-fd44a8eacbb

It is also used in other tests.